.github/workflows/checks.yml

```diff
@@ -0,0 +1,110 @@
+name: Basic checks
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - 1.6.x # remove when we create 'main'
+    tags-ignore: [ v.* ]
+
+jobs:
+  check-code-style:
+    name: Check Code Style
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Checkout GitHub merge
+        if: github.event.pull_request
+        run: |-
+          git fetch origin pull/${{ github.event.pull_request.number }}/merge:scratch
+          git checkout scratch
```

Comment:
Do you do the merge dance to mimic Travis' behaviour? (We do not do that in e.g. akka/akka.)

Reply:
I think I copied it from akka-projection, so we do in some projects at least.

Reply:
I will remove it. The simpler the better.